### ISSUE TITLE
Invalid anonymous class formating.

### ISSUE THREAD

**Issue by bweston92** [FriendsOfPHP/PHP-CS-Fixer#2030]
Anonymous class seems to loose indentation.

```
   1) unit/App/Remote/MessageSerializerTest.php
      ---------- begin diff ----------
      --- Original
      +++ New
      @@ @@
               $message = new class extends DomainMessage
      -        {
      -            
      -            public function __construct()
      -            {
      -                $this->init();
      -            }
      -            
      -            public function payload()
      -            {
      -                return [];
      -            }
      -            
      -            protected function setPayload(array $payload)
      -            {
      -                //
      -            }
      -            
      -            public function messageType()
      -            {
      -                return 'Test';
      -            }
      -        };
      + {
      +
      +     public function __construct()
      +     {
      +         $this->init();
      +     }
      +
      +     public function payload()
      +     {
      +         return [];
      +     }
      +
      +     protected function setPayload(array $payload)
      +     {
      +         //
      +     }
      +
      +     public function messageType()
      +     {
      +         return 'Test';
      +     }
      + };
       
               $SUT = new MessageSerializer();
               $SUT->messageToArray($message);
           }
       }
       
      
      ---------- end diff ----------
```

Config

```
<?php
$finder = Symfony\CS\Finder\DefaultFinder::create()
    ->in(__DIR__ . '/src')
    ->in(__DIR__ . '/tests');
$config = Symfony\CS\Config\Config::create();
$config->level(null);
$config->fixers(
    array(
        'braces',
        'duplicate_semicolon',
        'elseif',
        'empty_return',
        'encoding',
        'eof_ending',
        'function_call_space',
        'function_declaration',
        'indentation',
        'join_function',
        'line_after_namespace',
        'linefeed',
        'lowercase_constants',
        'lowercase_keywords',
        'parenthesis',
        'multiple_use',
        'method_argument_space',
        'object_operator',
        'php_closing_tag',
        'remove_lines_between_uses',
        'single_line_after_imports',
        'short_array_syntax',
        'short_tag',
        'standardize_not_equal',
        'trailing_spaces',
        'unused_use',
        'visibility',
        'whitespacy_lines',
    )
);
$config->finder($finder);
return $config;
```

Version of `php-cs-fixer`
```
vendor/bin/php-cs-fixer --version
PHP CS Fixer version 1.11.4 by Fabien Potencier
```

**Comment by GrahamCampbell** [comment]
This is fixed in the latest dev versions.

**Comment by bweston92** [comment]
@GrahamCampbell ok, thanks.

**Comment by keradus** [comment]
@GrahamCampbell you mean 2@dev or 1@dev ?
After great work of handling PHP7 on 1.11 by @SpacePossum I was still able to find out some new cases thanks to integration tests, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2017

**Comment by keradus** [comment]
Fix in #2037

**Comment by keradus** [comment]
fix is merged.
